backend/app/services/chat/config/shell_checker.py

```
@@ -160,3 +160,81 @@ def get_team_first_bot_shell_type(db: Session, team: Kind) -> str:
+
+    first_member = team_crd.spec.members[0]
+
+    # Get first bot
+    bot = (
+        db.query(Kind)
+        .filter(
+            Kind.user_id == team.user_id,
+            Kind.kind == "Bot",
+            Kind.name == first_member.botRef.name,
+            Kind.namespace == first_member.botRef.namespace,
+            Kind.is_active == True,
```

Ruff (0.14.14)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)